### PR TITLE
(done) batteries glow

### PR DESCRIPTION
**Pull request by Arkanic**
i think everyone gets the idea by now

### CODE REVIEW

File: core/src/mindustry/world/blocks/power/Battery.java [hunk 1]
```diff
@@ -1,10 +1,35 @@
...
     public Battery(String name){
         super(name);
         outputsPower = true;
         consumesPower = true;
     }
+    
+    @Override
+    public void load(){
+        super.load();
+        lightsRegion = Core.atlas.find(name + "-lights");
```
Comment (Quezler) [inline core/src/mindustry/world/blocks/power/Battery.java]: does this texture exist?

Comment (Arkanic) [reply core/src/mindustry/world/blocks/power/Battery.java]: no, haven't added sprites yet

~~try looking at the past 5 pulls~~

Comment (Arkanic) [reply core/src/mindustry/world/blocks/power/Battery.java]: i am still yet to pack the sprites

**Comment by Anuken** [comment]
image
oh no

**Comment by Arkanic** [comment]
maybe batteries where never meant to be

**Comment by Arkanic** [comment]
i have an idea
the shading is rendered below the battery, and the light hole is transparent
ill add a `-icon-base.png` sprite which effectively is just an orange background for the sprite

**Comment by Arkanic** [comment]
good idea in theory
i'll try again sometime soon

**Pull request closed without merging**